visualize-admin/visualization-tool
# Feat/improve chart tab titles

### **This PR is to optimize the UX for adding and editing tab titles**

**Issue**
1. Go to this [Link](https://visualization-tool-5nwbb9kd7-ixt1.vercel.app/create/X27AKmYCEzgl?dataSource=Prod)
2. Add a second Chart 
3. Click on any Chart
4. Notice how Add Label Button opens Add Label Options
5. Scroll down to "Title & Description" Section Notice there is no way to edit tab label

**How to test?**
1. Go to this [Link](https://visualization-tool-fdkin1fbd-ixt1.vercel.app/create/uM3BIxyxSCxv?dataSource=Prod)
2. Add a second Chart
3. Click on any Chart
4. Notice how Chart Changes without opening Tab Label Editor 
5. Click Dropdown Arrow
6. See new Edit tab Label - Click on it and edit label
7. Scroll down to "Title & Description" Section Notice you can edit tab title here as well.
